code/datums/components/connect_mob_behalf.dm

```diff
@@ -0,0 +1,59 @@
+/// This component behaves similar to connect_loc_behalf, but working off clients and mobs instead of loc
+/// To be clear, we hook into a signal on a tracked client's mob
+/// We retain the ability to react to that signal on a seperate listener, which makes this quite powerful
+/datum/component/connect_mob_behalf
```

Comment:
Is this used in this PR?

Reply:
> Was expecting violent conflicts with my parallax subsystem

And this is why you are not seeing conflicts, because this was what lemon hooked into the parallax system



> Is this used in this PR?

It isn't. I was hoping we could utilize it somehow despite TG and Bee's code differences, but I am not knowledgable enough about parallax overall to say whether we can or not.

As you said, TG's is based more upon the the mob and is not rate-limited compared to us.

I just wanted to provide a launching point if anyone who knew enough could have an easier time hooking it in, since Lemon made some serious improvements and I really like their spacedrifting.